R/cqcp_filter.R

```diff
@@ -369,14 +369,20 @@ cqcp_m5 <- function(data, radius = 3000, n_buddies = 5, alpha = 0.1,
   dist <- dist[(p_x != p_y) & (!is.na(distance))] # reduce table size
   combi <- dist[distance <= radius] # station within radius
   
+  # check for p_id column type (integer?) and convert indices to integer, if necessary
+  if (is.integer(data$p_id)) {
```

Comment:
isn't this supposed to be `if (!is.integer(data$p_id))` or what is `p_x` or `p_y` exactly?

Reply:
No, actually correct. If `data$p_id` is of type integer, the two indices `p_x` and `p_y` need to be integer as well. Yet they are are of type character originally.